### PR TITLE
section 10 faq

### PR DESCRIPTION
**Pull request by Dilekcn**

### CODE REVIEW

File: client/src/pages/FrequentlyAskedQuestions/FrequentlyAskedQuestions.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.faQuestions_container {
```
Comment (ewaDev) [inline client/src/pages/FrequentlyAskedQuestions/FrequentlyAskedQuestions.css:1]: please keep naming consistent - in the other file you use faq_container

File: client/src/pages/Home/Faq/Faq.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    text-decoration: none;
+}
+.faq_button:hover{
+    background-color: var(--darkgreen-color);
+}
+
+.faq_button_tooltip {
+	font-family: 'Ambit SemiBold';
+	font-size: 22px;
+    background-color:var(--blue-color) ;
```
Comment (ewaDev) [inline client/src/pages/Home/Faq/Faq.css:30]: please fix indentation

File: client/src/pages/Home/Faq/Faq.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+import "./Faq.css";
+
+function Faq() {
+	const [isShown, setIsShown] = useState(false);
+
+	return (
+		<div className="faq_container">
+			<div className="faq_button">
+				<Link
+                to="/faq"
```
Comment (ewaDev) [inline client/src/pages/Home/Faq/Faq.js:12]: is this the correct indentation?

**Review by ewaDev — CHANGES_REQUESTED** [review]

File: client/src/pages/FrequentlyAskedQuestions/FrequentlyAskedQuestions.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "./FrequentlyAskedQuestions.css";
+
+function FrequentlyAskedQuestions() {
+  return <div className='faQuestions_container'>
+      <h1>Frequently Asked Questions</h1>
+
```
Comment (khaledkz) [inline client/src/pages/FrequentlyAskedQuestions/FrequentlyAskedQuestions.js:7]: can you remove this empty line

File: client/src/pages/Home/Faq/Faq.css (hunk 2 above)
Comment (Dilekcn) [reply client/src/pages/Home/Faq/Faq.css:30]: indentation fixed

File: client/src/pages/Home/Faq/Faq.js (hunk 3 above)
Comment (Dilekcn) [reply client/src/pages/Home/Faq/Faq.js:12]: indentation fixed

**Review by EwaGrabowiecka — APPROVED** [review]

**Review by khaledkz — CHANGES_REQUESTED** [review]
Can you fix the merge conflict

**Review by khaledkz — APPROVED** [review]

**Review by ewaDev — APPROVED** [review]

**Pull request merged**